src/vm/debugdebugger.cpp

```diff
@@ -512,6 +517,10 @@ FCIMPL4(void, DebugStackTrace::GetStackFramesInternal,
             I4 *pILI4 = (I4 *)((I4ARRAYREF)pStackFrameHelper->rgiILOffset)->GetDirectPointerToNonObjectElements();
             pILI4[iNumValidFrames] = data.pElements[i].dwILOffset;
 
+            // Assembly
+            OBJECTREF pAssembly = pFunc->GetAssembly()->GetExposedObject(); 
```

Comment:
This does not handle generics properly. You want to keep reference to LoaderAllocator here, not to the Assembly object.

Reply:
`rgMethodHandle` may be the instantiated method - so you need to keep all the generic instantiations constituents alive. The easiest way to do that is to just keep the loader allocator alive.